pkg/config/generate.go

```diff
@@ -388,3 +395,17 @@ func Generate(cs *api.OpenShiftManagedCluster, pluginConfig api.PluginConfig) (e
 
 	return
 }
+
+func certEqual(certNew, certOld *x509.Certificate) bool {
+	if certOld == nil {
+		return false
+	}
+	if !reflect.DeepEqual(certNew.Subject, certOld.Subject) ||
+		!reflect.DeepEqual(certNew.DNSNames, certOld.DNSNames) ||
+		!reflect.DeepEqual(certNew.ExtKeyUsage, certOld.ExtKeyUsage) ||
+		!reflect.DeepEqual(certNew.IPAddresses, certOld.IPAddresses) ||
+		!reflect.DeepEqual(certNew.Issuer, certOld.Issuer) {
```

Comment:
Issuer is just a name.  This function will not fire correctly if the CA certificate is reissued and its name doesn't change.  That's why we use CheckSignatureFrom above.  You need to do the same.